compatibility/compatibility-vec12to20/src/main/java/com/foursoft/harness/compatibility/vec12to20/Vec12XTo20XClassMapper.java

```diff
@@ -70,10 +67,12 @@ public Vec12XTo20XClassMapper() {
 
     @Override
     public Class<?> map(final Class<?> clazz) {
-        final Class<?> aClass = classMap.get(clazz);
-        return aClass != null
-                ? aClass
-                : classMap.getOrDefault(ClassUtils.getNonProxyClass(clazz), super.map(clazz));
+        return resolveExplicitClassMappings(clazz).orElseGet(() -> super.map(clazz));
+    }
+
+    private Optional<Class<?>> resolveExplicitClassMappings(final Class<?> clazz) {
+        return Optional.<Class<?>>ofNullable(explicitClassMappings.get(clazz))
+                .or(() -> Optional.ofNullable(explicitClassMappings.get(ClassUtils.getNonProxyClass(clazz))));
```

Comment:
@DManstrator extract functional units into small methods, give them consice names. Then you don't need inline comments.